### PR TITLE
docs(installation/dcos) updated to DCOS 1.9 and added a demo app example

### PR DESCRIPTION
**Pull request by shashiranjan84**

### CODE REVIEW

File: app/install/dcos.md [hunk 1]
```diff
@@ -92,11 +94,45 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
...
+    $ curl marathon-lb.marathon.mesos:10001
+    $ curl marathon-lb.marathon.mesos:10002
     ```
 
-7. **Using Kong**
+7. **Deploy an upstream server**
+
+    For this demo, we created an app which returns `Hello world` on port `8080`.
+    Using the `my_app.json` file from the kong-dist-dcos repo, deploy the app in
+    the cluster which will act as a backend server to process request received
```
Comment (theetrain) [inline app/install/dcos.md]: > ... to process `requests` received ...

Comment (shashiranjan84) [inline app/install/dcos.md]: @theetrain thanks, updated.

File: app/install/dcos.md [hunk 2]
```diff
@@ -75,8 +76,9 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
 
 5. **Deploy Kong**
 
-    Using the `kong_<postgres|cassandra>.json` file from the
-    kong-dist-mesos repo, deploy Kong in the cluster:
+    Using the `kong_<postgres|cassandra>.json` file from the kong-dist-dcos
```
Comment (nijikokun) [inline app/install/dcos.md]: Backticks around `kong-dist-dcos`

File: app/install/dcos.md [hunk 3]
```diff
@@ -75,8 +76,9 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
...
+    repo, deploy Kong in the cluster. Template registers Kong's `proxy` and
+    `admin`ports on Marathon-LB as service ports `10001` and `10002` respectively:
```
Comment (nijikokun) [inline app/install/dcos.md]: space between `admin` and ports

Comment (nijikokun) [inline app/install/dcos.md]: Also, should `Marathon-LB` be formatted this way? Should it be `Marathan Loadbalancer`?

File: app/install/dcos.md [hunk 4]
```diff
@@ -92,11 +94,45 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
...
-    $ curl marathon-lb.maraton.mesos:10001
-    $ curl marathon-lb.maraton.mesos:10002
+    $ curl marathon-lb.marathon.mesos:10001
+    $ curl marathon-lb.marathon.mesos:10002
     ```
 
-7. **Using Kong**
+7. **Deploy an upstream server**
+
+    For this demo, we created an app which returns `Hello world` on port `8080`.
```
Comment (nijikokun) [inline app/install/dcos.md]: `we will create` as you have not yet done the demo

File: app/install/dcos.md [hunk 5]
```diff
@@ -92,11 +94,45 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
...
+    Using the `my_app.json` file from the kong-dist-dcos repo, deploy the app in
```
Comment (nijikokun) [inline app/install/dcos.md]: Backticks around `kong-dist-dcos`

File: app/install/dcos.md [hunk 6]
```diff
@@ -43,30 +44,30 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
...
-3. **Deploy Marathon-lb**
+3. **Deploy Marathon-LB**
 
-    Using the `marathon-lb-internal.json` file included in the kong-dist-mesos
-    repo, deploy Marathon-lb for internal and external service discovery
-    support:
+    We will use [Marathon-LB]({{ page.links.marathon-lb }}) for load balancing
+    N/S traffic to cluster and [VIPs]({{page.links.vips}}) for load balancing
+    E/W traffic. Using the package `marathon-lb` deploy Marathon-LB for
+    external load blance support and service discovery:
```
Comment (theetrain) [inline app/install/dcos.md]: `balance`

Comment (nijikokun) [inline app/install/dcos.md]: The above paragraph doesn't make sense.

> We will use Marathon-LB .... Using the package

Needs a sentence break to introduce the package after denoting usage.

``` We will use the `marathon-lb` package to deploy [Marathon-LB]({{ page.links.marathon-lb }}) for load balancing,  and N/S traffic to cluster and [VIPs]({{page.links.vips}}) for load balancing E/W traffic.```

File: app/install/dcos.md (hunk 3 above)
Comment (shashiranjan84) [inline app/install/dcos.md]: I see space, I don't know why its showing like that.

File: app/install/dcos.md (hunk 2 above)
Comment (shashiranjan84) [inline app/install/dcos.md]: That how its was published 1st and reviewed by @thibaultcha. I don't backticks needes here

File: app/install/dcos.md [hunk 7]
```diff
@@ -92,11 +96,45 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
...
+
+8. **Using Kong**
+    
+    Create an API on Kong:
+
+    ```bash
+    $ curl -i -X POST marathon-lb.marathon.mesos:10002/apis \
+    --data "name=myapp" \
+    --data "hosts=myapp.com" \
+    --data "upstream_url=http://myapp.marathon.l4lb.thisdcos.directory:8080"
```
Comment (thibaultcha) [inline app/install/dcos.md]: One last thing: all of our shell commands examples in this website use a 4 space indentation on `--data` and other `curl` arguments.

File: app/install/dcos.md [hunk 8]
```diff
@@ -92,11 +96,45 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
...
+    HTTP/1.1 201 Created
+    ...
+
+    ```
+
+    Make a request to the API:
+
+    ```bash
+    $ curl -i -X GET marathon-lb.marathon.mesos:10001 \
+    --header "Host:myapp.com"
```
Comment (thibaultcha) [inline app/install/dcos.md]: ditto

Comment (shashiranjan84) [inline app/install/dcos.md]: @thibaultcha done.

**Comment by theetrain** [comment]
FYI, ignore the travis build errors as Travis is experiencing issues: https://twitter.com/traviscistatus/status/859505575465426944

**Pull request merged**